.NET/Microsoft.Recognizer.Text.TimeZone/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
This file shouldn't exist in the repo as it's auto-generated by the build process when targeting different frameworks.